tfe/resource_tfe_registry_module.go

```diff
@@ -59,39 +67,109 @@ func resourceTFERegistryModule() *schema.Resource {
 					},
 				},
 			},
+			"namespace": {
+				Type:         schema.TypeString,
+				Computed:     true,
```

Comment:
I don't think this, or registry_name, or module_provider attributes are Computed any longer since they are input arguments now. Computed is usually reserved for attributes that the user cannot set and are provided by the API (like 'created' timestamps)